azure-pipelines.yml

```diff
@@ -367,7 +367,8 @@ jobs:
         kind: maui_scenarios_android
         projectFile: maui_scenarios_android.proj
         dotnetVersionsLinks:
-          8.0: https://aka.ms/dotnet/sdk/maui/net8.0.json # Use 8.0 key and net8.0 link until Maui is ready for net9.0
+          9.0: https://maui.blob.core.windows.net/metadata/sdks/net9.0.json
```

Comment:
@mattleibow can someone create the aka.ms links for these? I don't think I can create them without "owning" the URL.